lib/ansible/modules/cloud/openstack/os_network.py

```diff
@@ -155,7 +163,8 @@ def main():
         provider_network_type=dict(required=False),
         provider_segmentation_id=dict(required=False),
         state=dict(default='present', choices=['absent', 'present']),
-        project=dict(default=None)
+        project=dict(default=None),
+        port_security_enabled=dict(default=None, type='bool')
```

Comment:
Should this default to `False` instead of `None` to match the default in openstacksdk?

Reply:
Yes, it should be `False`

Reply:
@tbreeds could you make that small fix so it can be merged please!